src/detect-icmp-seq.c

```diff
@@ -366,11 +362,9 @@ static int DetectIcmpSeqParseTest03 (void)
 {
     DetectIcmpSeqData *iseq = NULL;
```

Comment:
in your next PR, could you change these to a single line, e.g.
`DetectIcmpSeqData *iseq = DetectIcmpSeqParse(NULL, "badc");`